### PR TITLE
V2 Deploy Part 1

### PR DESCRIPTION
**Pull request by Joeysantoro**
This PR deploys everything in V2 except the FEI PCV wrappers which will come in a subsequent PR

Also fixes a bug in the UniswapPCVDeposit resistant balance calculation where the result is not scaled by the proportion held by the contract

- [x] seed TRIBE to LBP stabilizer
- [ ] check that invertOracle direction is correct for TribeReserveStabilizer

### CODE REVIEW

File: contract-addresses/mainnetAddresses.json [hunk 1]
```diff
@@ -77,6 +77,10 @@
     "proxyAdminAddress" : { "artifact" : "ProxyAdmin", "address": "0xf8c2b645988b7658E7748BA637fE25bdD46A704A"},
     "raiAddress": {"artifact" : "IERC20", "address" : "0x03ab458634910AaD20eF5f1C8ee96F1D6ac54919"},
     "raiBondingCurveAddress": { "artifact": "BondingCurve", "address":  "0x25d60212D47Dd8F6Ff0469367E4c6C98Cd3411A5"},
+    "rariPool19DpiAddress": { "artifact" : "CErc20Delegator", "address": "0xf06f65a6b7d2c401fcb8b3273d036d21fe2a5963"},
```
Comment (Joeysantoro) [inline contract-addresses/mainnetAddresses.json:80]: Added these copies of the indexCoop and poolParty PCV deposits which adhere to naming conventions

File: end-to-end/e2e.spec.ts [hunk 2]
```diff
@@ -436,71 +436,6 @@ describe('e2e', function () {
     })
   });
 
-  describe('Reweights', async () => {
```
Comment (Joeysantoro) [inline end-to-end/e2e.spec.ts:439]: Deleted reweights from e2e which are not a part of v2

File: end-to-end/proposals_config.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
-    "balancerLBP": {
+    "v2Phase1": {
         "deploy" : true,
         "exec" : false,
         "proposerAddress" : "0xe0ac4559739bD36f0913FB0A3f5bFC19BCBaCD52",
         "voterAddress" : "0xB8f482539F2d3Ae2C9ea6076894df36D1f632775"
     },
-    "upgrade" : {
+    "fip_10b": {
```
Comment (Joeysantoro) [inline end-to-end/proposals_config.json:8]: fip 10b is the curve deposit which is currently separate from the main phase 1 deploy

File: scripts/utils/sudo.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ async function sudo(addresses, logging = false) {
   await core.grantBurner(accounts[0], {from: accounts[0]});
 
   logging ? console.log('Minting FEI to accounts[0]') : undefined;
-  await fei.mint(accounts[0], new BN('10000000000000000000000000000000000'));
+  await fei.mint(accounts[0], new BN('10000000000000000000000000'));
```
Comment (Joeysantoro) [inline scripts/utils/sudo.js:41]: this was messing up the CR oracle calculation so I lowered amount to 1M FEI

**Comment by ditchfieldcaleb** [comment]
@Joeysantoro CI is showing two failed tests - did you have a look at them?

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol [hunk 5]
```diff
@@ -155,17 +155,26 @@ contract UniswapPCVDeposit is IUniswapPCVDeposit, PCVDeposit, UniRef {
         5. rETH = sqrt(k / P)
         
         and rFEI = k / rETH by 1.
+
+        Finally scale the resistant reserves by the ratio owned by the contract
      */
     function resistantBalanceAndFei() public view override returns(uint256, uint256) {
-        (uint256 feiInPool, uint256 ethInPool) = getReserves();
-        Decimal.D256 memory ethFei = readOracle();
+        (uint256 feiInPool, uint256 otherInPool) = getReserves();
```
Comment (ditchfieldcaleb) [inline contracts/pcv/uniswap/UniswapPCVDeposit.sol:162]: Is the order of the addresses retrieved here always going to have fei first?

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol [hunk 6]
```diff
@@ -155,17 +155,26 @@ contract UniswapPCVDeposit is IUniswapPCVDeposit, PCVDeposit, UniRef {
...
+
+        Decimal.D256 memory price = readOracle();
```
Comment (ditchfieldcaleb) [inline contracts/pcv/uniswap/UniswapPCVDeposit.sol:164]: change to priceOf(Token?) for better readability

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol [hunk 7]
```diff
@@ -155,17 +155,26 @@ contract UniswapPCVDeposit is IUniswapPCVDeposit, PCVDeposit, UniRef {
...
-        Decimal.D256 memory k = Decimal.from(feiInPool).mul(ethInPool);
-        uint256 resistantEthInPool = k.div(ethFei).asUint256().sqrt();
-        uint256 resistantFeiInPool = k.div(resistantEthInPool).asUint256();
+        // resistant other/fei in pool
+        uint256 resistantOtherInPool = Decimal.one().div(price).mul(k).asUint256().sqrt();
 
-        return (resistantEthInPool, resistantFeiInPool);
+        uint256 resistantFeiInPool = Decimal.ratio(k, resistantOtherInPool).asUint256();
+
+        Decimal.D256 memory ratioOwned = _ratioOwned();
```
Comment (ditchfieldcaleb) [inline contracts/pcv/uniswap/UniswapPCVDeposit.sol:173]: I haven't used the decimal library enough to know this - is this actually a ratio or is this a percentage? Just want to make sure we aren't multiplying by >100% of the token values.

**Review by ditchfieldcaleb — COMMENTED** [review]
First pass appears fine, though see the couple of comments I made.
I'm going to do a second pass and poke at it with VsCode some.

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol (hunk 5 above)
Comment (Joeysantoro) [reply contracts/pcv/uniswap/UniswapPCVDeposit.sol:162]: Yes

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol (hunk 7 above)
Comment (Joeysantoro) [reply contracts/pcv/uniswap/UniswapPCVDeposit.sol:173]: Its a percentage. Ratio as in fraction

File: contracts/pcv/uniswap/UniswapPCVDeposit.sol (hunk 6 above)
Comment (Joeysantoro) [reply contracts/pcv/uniswap/UniswapPCVDeposit.sol]: Done

**Comment by Joeysantoro** [comment]
> @Joeysantoro CI is showing two failed tests - did you have a look at them?

One was just an env failure, fixed the other

**Review by ditchfieldcaleb — APPROVED** [review]

**Pull request merged**